doc/classes/Line2D.xml

```diff
@@ -210,44 +213,63 @@
 	</methods>
 	<members>
 		<member name="begin_cap_mode" type="int" setter="set_begin_cap_mode" getter="get_begin_cap_mode" enum="Line2D.LineCapMode">
+			The style used for the shape in the first point of the line.
 		</member>
 		<member name="default_color" type="Color" setter="set_default_color" getter="get_default_color">
+      The color of the Line2D. Will not be used if a gradient is set.
 		</member>
 		<member name="end_cap_mode" type="int" setter="set_end_cap_mode" getter="get_end_cap_mode" enum="Line2D.LineCapMode">
+			The style used for the shape in the last point of the line.
 		</member>
 		<member name="gradient" type="Gradient" setter="set_gradient" getter="get_gradient">
+			The gradient is drawn through the whole line from start to finish. The default color will not be used if a gradient is set.
 		</member>
 		<member name="joint_mode" type="int" setter="set_joint_mode" getter="get_joint_mode" enum="Line2D.LineJointMode">
+			The style for the points inbetween the start and the end.
 		</member>
 		<member name="points" type="PoolVector2Array" setter="set_points" getter="get_points">
+			The points that form the lines. The line is drawn between every point set in this array.
 		</member>
 		<member name="round_precision" type="int" setter="set_round_precision" getter="get_round_precision">
+			Sets how smooth the rounded joints and caps will be. This is only used if a cap or joint is set as round.
```

Comment:
Don't use sets/gets for member variables - they do both.  "Smoothness" or "Smoothing" might work.